modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,243 +53,371 @@ and device ID 0x1015
+<4> The node selector to select which node to be configured. User can choose to
+label the nodes manually or with tools like Kubernetes Node Feature Discovery.
+Only SR-IOV network devices on selected nodes will be configured. And the SR-IOV
+CNI plugin and device plugin will be only deployed on selected nodes.
+<5> The priority of the policy,  the larger number gets lower priority.
+Range from 0 to 99.
+<6> The MTU of the virtual functions. Range from 1 to 9000. Leave it blank if
+you don't need to change the MTU.
+<7> The number of the virtual functions for each SR-IOV network device.
+<8> The vendor hex code of SR-IoV device. Allowed value "8086", "15b3".
+<9> The device hex code of SR-IoV device. Allowed value "158b", "1015", "1017".
+<10> The names of SR-IoV physical function.
+<11> The PCI addresses of SR-IoV physical function.
+<12> The driver type of the virtual functions. Allowed value "netdevice",
+"vfio-pci". Defaults to "netdevice".
+<13> The RDMA mode. Defaults to false.
 
-. Run the following command to create the `sriov-conf` CR:
-+
+== Configuring SR-IOV networks
+
+The SR-IOV Network Operator also introduces a CRD `SriovNetwork` for creating
+the Custom Resource of `NetworkAttachmentDefinition` of SR-IOV CNI plugin. When
+you create a CR of `SriovNetwork`, the operator will created a CR of
+`NetworkAttachmentDefinition` according.
+
+You should never have to modify this CRD. Instructions for creating or 
+modifying a CR are provided in this documentation as appropriate.
+
+[NOTE]
+=====
+You shall not modify or delete a Custom `SriovNetwork`, when it has been used by
+any running Pods.
+=====
+
+The following is an example of a typical Custom Resource for
+`SriovNetworkNodePolicy`.
+
+[source,yaml]
 ----
-$ oc create -f sriov-conf.yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-conf <1>
+  namespace: sriov-network-operator <2>
+spec:
+  networkNamespace: default <3>
+  ipam: | <4>
+    {
+      "type": "host-local",
+      "subnet": "10.56.217.0/24",
+      "rangeStart": "10.56.217.171",
+      "rangeEnd": "10.56.217.181",
+      "routes": [{
+        "dst": "10.0.0.0/8"
+      }],
+      "gateway": "10.56.217.1"
+    }
+  vlan: 0 <5>
+  resourceName: sriov <6>
+  spoofChk: true
+  trust: false
 ----
+<1> The name of the CR. The generated `NetworkAttachmentDefinition` CR will use
+the same name.
+<2> The namespace of the CR, this must be in the same namespace of operator.
+<3> The namespace where the `NetworkAttachmentDefinition` CR will be created.
+<4> The IPAM configuration for SR-IOV CNI plugin.
+<5> The VLAN ID for SR-IOV CNI plugin. Range from 0 to 4095, default to 0.
+<6> The SRIOV Network device plugin endpoint resource name. It shall matches
+the `resourceName` defined in the `SriovNetworkNodePolicy` CR.
+<7> The Virtual Function spoof check. Boolean, default to false.
```

Comment:
These two config options `spoofChk` and `trust` are not supported by sriov-cni in 4.2, perhaps better to not exposing these options to user in the doc.

Reply:
Fixed.